assets/css/sensei-course-theme.scss

```diff
@@ -2,10 +2,32 @@
 @import './sensei-course-theme/fonts';
 
 .sensei-course-theme {
+	--primary-color: var(--theme-color, #30968B);
```

Comment:
Just a notice that we'll have some style conflicts in the next PRs, like this https://github.com/Automattic/sensei/pull/4430/files#diff-3ce848a738a931b14431516f745c6578e71cd8ec5229a994e7254f505fd54fbdR3

And here: https://github.com/Automattic/sensei/pull/4433/files#diff-f20c91303932abad7b1468e35ea93c6610209df38bc418a407f201db93889587R1 (file moved)